sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/SaveAsHiveFile.scala

```diff
@@ -217,12 +217,7 @@ private[hive] trait SaveAsHiveFile extends DataWritingCommand {
     val inputPathUri: URI = inputPath.toUri
     val inputPathName: String = inputPathUri.getPath
     val fs: FileSystem = inputPath.getFileSystem(hadoopConf)
-    var stagingPathName: String =
```

Comment:
This code is introduced form https://github.com/apache/spark/pull/12770, do you check the relative UT? Maybe UT relating with SessionState and DDLSuite.

Reply:
Yes, I had checked the pr [#12770](https://github.com/apache/spark/pull/12770/files) too. But I'm not very clear why we should do like that.